### PR TITLE
Minor: updated language to reflect change to .md

### PR DESCRIPTION
**Pull request by estelle**
#### Metadata

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [ ] Fixes a typo, bug, or other error

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -364,19 +364,17 @@ Adding a new document is relatively straightforward, especially if you can
...
-- Documents can be authored in either Markdown or HTML. However, we're converting
-  the site to Markdown one section at a time, and don't want to mix authoring
-  formats within a section. At this point we have only converted the JavaScript
-  documentation. So if you are adding a new document under
-  `files/en-us/web/javascript`, make it a Markdown file. Otherwise, make it an
-  HTML file.
+- Documents can be authored in either Markdown or HTML. However, we've converted
+  the site to Markdown and don't want to mix authoring
+  formats within a section. If you are adding a new document under
+  `files/en-us/web/`, make it a Markdown file.
```
Comment (teoli2003) [inline README.md:370]: Even if HTML files are still supported, we have converted the whole website to MD.
```suggestion
- Documents must be authored in Markdown.
```

File: README.md [hunk 2]
```diff
@@ -397,8 +395,8 @@ As we outlined above, the step-by-step process in general would be:
     git checkout -b my-add
     ```
 
-1. Create one or more new document folders, each with their own `index.html`
-or `index.md` file.
+1. Create one or more new document folders, each with their own `index.md`
+(or `index.html`) file.
```
Comment (teoli2003) [inline README.md:399]:
```suggestion
1. Create one or more new document folders, each with their own `index.md` file.
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]
Even if we technically support HTML files,  the whole site is now in Markdown. I don't think we need to complexify the prose here to explain something that will lead to PRs being rejected.

And: good catch! we completely forgot about this file! Much needed!

**Comment by teoli2003** [comment]
@schalkneethling There is a failing test here: the action doesn't find `markdownlint-cli`. I would ignore it, but I think this need to be fixed? Before opening an issue, is this something new? Or only applied to a few files?

**Comment by schalkneethling** [comment]
> @schalkneethling There is a failing test here: the action doesn't find `markdownlint-cli`. I would ignore it, but I think this need to be fixed? Before opening an issue, is this something new? Or only applied to a few files?

@teoli2003 It does not find it, but then goes ahead and installs it, there actual error is on the next line:

```
npm WARN exec The following package was not found and will be installed: markdownlint-cli
README.md:370:47 MD009/no-trailing-spaces Trailing spaces [Expected: 0 or 2; Actual: 1] # this here
```

**Review by teoli2003 — APPROVED** [review]
Thank you, @estelle

**Pull request merged**